.nais/nais-dev.yaml

```diff
@@ -8,6 +8,8 @@ metadata:
 spec:
   image: {{image}}
   port: 8080
+  ingresses:
+    - https://amt-arena-acl.intern.dev.nav.no

```

Comment:
Trengs ikke det samme i prod også?

Reply:
Nei, appene bruker service discovery. Jeg trengte denne for å teste i dev fra postman. Vurderte å fjerne den, men tenkte det var kjekt å ha for testing i dev.

Reply:
Aha, i dag lært jeg 👍